app/src/main/java/com/spoony/spoony/core/designsystem/theme/Type.kt

```diff
@@ -1,34 +1,276 @@
+    var title2sb by mutableStateOf(title2sb)
+        private set
+    var body1b by mutableStateOf(body1b)
+        private set
+    var body1sb by mutableStateOf(body1sb)
+        private set
+    var body1m by mutableStateOf(body1m)
+        private set
+    var body2b by mutableStateOf(body2b)
+        private set
+    var body2sb by mutableStateOf(body2sb)
+        private set
+    var body2m by mutableStateOf(body2m)
+        private set
+    var caption1b by mutableStateOf(caption1b)
+        private set
+    var caption1m by mutableStateOf(caption1m)
+        private set
+    var caption2b by mutableStateOf(caption2b)
+        private set
+    var caption2m by mutableStateOf(caption2m)
+        private set
+
+    fun copy(
+        title1: TextStyle = this.title1,
+        title2b: TextStyle = this.title2b,
+        title2sb: TextStyle = this.title2sb,
+        body1b: TextStyle = this.body1b,
+        body1sb: TextStyle = this.body1sb,
+        body1m: TextStyle = this.body1m,
+        body2b: TextStyle = this.body2b,
+        body2sb: TextStyle = this.body2sb,
+        body2m: TextStyle = this.body2m,
+        caption1b: TextStyle = this.caption1b,
+        caption1m: TextStyle = this.caption1m,
+        caption2b: TextStyle = this.caption2b,
+        caption2m: TextStyle = this.caption2m,
+
+    ): SpoonyTypography = SpoonyTypography(
+        title1,
+        title2b,
+        title2sb,
+        body1b,
+        body1sb,
+        body1m,
+        body2b,
+        body2sb,
+        body2m,
+        caption1b,
+        caption1m,
+        caption2b,
+        caption2m,
     )
-    /* Other default text styles to override
-    titleLarge = TextStyle(
-        fontFamily = FontFamily.Default,
-        fontWeight = FontWeight.Normal,
-        fontSize = 22.sp,
-        lineHeight = 28.sp,
-        letterSpacing = 0.sp
-    ),
-    labelSmall = TextStyle(
-        fontFamily = FontFamily.Default,
-        fontWeight = FontWeight.Medium,
-        fontSize = 11.sp,
-        lineHeight = 16.sp,
-        letterSpacing = 0.5.sp
+
+    fun update(other: SpoonyTypography) {
+        title1 = other.title1
+        title2b = other.title2b
+        title2sb = other.title2sb
+        body1b = other.body1b
+        body1sb = other.body1sb
+        body1m = other.body1m
+        body2b = other.body2b
+        body2sb = other.body2sb
+        body2m = other.body2m
+        caption1b = other.caption1b
+        caption1m = other.caption1m
+        caption2b = other.caption2b
+        caption2m = other.caption2m
+    }
+}
+
+private fun SpoonyTextStyle(
+    fontFamily: FontFamily,
+    fontSize: TextUnit,
+    lineHeight: TextUnit,
+    letterSpacing: TextUnit
+): TextStyle = TextStyle(
+    fontFamily = fontFamily,
+    fontSize = fontSize,
+    lineHeight = lineHeight,
+    letterSpacing = letterSpacing,
+    lineHeightStyle = LineHeightStyle(
+        alignment = LineHeightStyle.Alignment.Center,
+        trim = LineHeightStyle.Trim.None
```

Comment:
```suggestion
private fun SpoonyTextStyle(
    fontFamily: FontFamily,
    fontSize: TextUnit,
    lineHeight: TextUnit = 1.45.em,
    letterSpacing: TextUnit = (-0.02).em
): TextStyle = TextStyle(
    fontFamily = fontFamily,
    fontSize = fontSize,
    lineHeight = lineHeight,
    letterSpacing = letterSpacing,
    lineHeightStyle = LineHeightStyle(
        alignment = LineHeightStyle.Alignment.Center,
        trim = LineHeightStyle.Trim.None
```

이렇게 디폴트 값을 넣는건 어떨까요??
아래 코드를 보니 모두 저 값이 디폴트로 들어가는 것 같아용

Reply:
좋은데요!! 수정하겠습니다!!